res/images/icons/scalable/apps/mixxx_macos.svg

```diff
@@ -0,0 +1,1570 @@
+      <rect
+         width="4"
+         height="1.0000005"
+         x="-36"
+         y="-16"
+         transform="matrix(0,-1,-1,0,0,0)"
+         id="rect3908"
+         style="color:#000000;display:inline;overflow:visible;visibility:visible;fill:#ffffff;fill-opacity:0.8243243;fill-rule:nonzero;stroke:none;stroke-width:0.9999997;marker:none;enable-background:accumulate" />
+      <rect
+         width="3.0000014"
+         height="1"
+         x="-35"
+         y="-20"
+         transform="matrix(0,-1,-1,0,0,0)"
+         id="rect3910"
+         style="color:#000000;display:inline;overflow:visible;visibility:visible;fill:#ffffff;fill-opacity:0.8243243;fill-rule:nonzero;stroke:none;stroke-width:0.9999997;marker:none;enable-background:accumulate" />
+      <rect
+         width="6"
+         height="1.0000005"
+         x="-38"
+         y="-24"
+         transform="matrix(0,-1,-1,0,0,0)"
+         id="rect3912"
+         style="color:#000000;display:inline;overflow:visible;visibility:visible;fill:#ffffff;fill-opacity:0.8243243;fill-rule:nonzero;stroke:none;stroke-width:0.9999997;marker:none;enable-background:accumulate" />
+      <rect
+         width="3.0000014"
+         height="1"
+         x="-35"
+         y="-28"
+         transform="matrix(0,-1,-1,0,0,0)"
+         id="rect3914"
+         style="color:#000000;display:inline;overflow:visible;visibility:visible;fill:#ffffff;fill-opacity:0.8243243;fill-rule:nonzero;stroke:none;stroke-width:0.9999997;marker:none;enable-background:accumulate" />
+      <rect
+         width="3.9999995"
+         height="1"
+         x="-36"
+         y="-32"
+         transform="matrix(0,-1,-1,0,0,0)"
+         id="rect3916"
+         style="color:#000000;display:inline;overflow:visible;visibility:visible;fill:#ffffff;fill-opacity:0.8243243;fill-rule:nonzero;stroke:none;stroke-width:0.9999997;marker:none;enable-background:accumulate" />
+      <rect
+         width="5.0000014"
+         height="1"
+         x="-37"
+         y="-35"
+         transform="matrix(0,-1,-1,0,0,0)"
+         id="rect3919"
+         style="color:#000000;display:inline;overflow:visible;visibility:visible;fill:#ffffff;fill-opacity:0.8243243;fill-rule:nonzero;stroke:none;stroke-width:0.9999997;marker:none;enable-background:accumulate" />
+      <rect
+         width="5.999999"
+         height="1"
+         x="-38"
+         y="-39"
+         transform="matrix(0,-1,-1,0,0,0)"
+         id="rect3921"
+         style="color:#000000;display:inline;overflow:visible;visibility:visible;fill:#ffffff;fill-opacity:0.8243243;fill-rule:nonzero;stroke:none;stroke-width:0.9999997;marker:none;enable-background:accumulate" />
+    </g>
+  </g>
+  <g
+     inkscape:groupmode="layer"
+     id="layer2"
+     inkscape:label="16x16 version"
+     style="display:none"
```

Comment:
hidden layers can be removed.
please save as "Plain SVG" to get rid of the Inkscape specific attributes.

Reply:
While I agree that hidden layers should be removed, the existing icon also has Inkscape-specific attributes, which are quite useful as they e.g. provide grid lines along which specific parts of the icon are aligned. Since the SVG version of this new icon is only used for development and not by the app bundle itself, I think keeping the Inkscape attributes around would be a good idea (it is easy enough to remove them in the future, if desired).

Reply:
Ok, let's keep the Inkscape version then